test/blas_test.hpp

```diff
@@ -99,6 +99,19 @@ inline cl::sycl::queue make_queue() {
   return queue;
 }
 
+/**
+ * @brief Generates a random scalar in the specified range
+ * @param rangeMin range minimum
+ * @param rangeMax range maximum
+ */
+template <typename scalar_t>
+static inline scalar_t random_scalar(scalar_t rangeMin, scalar_t rangeMax) {
```

Comment:
BLAS has `fill_random` function that accepts the vector and fills it. To unify all the BLAS data to follow the same random seed and random generator algorithm, can you make them align with each other. e,g  `fill_random` can use the random_scaler to populate each element and you can replace the seed generator here with what is  used in `fill_random`

Reply:
See https://github.com/codeplaysoftware/sycl-blas/pull/265#discussion_r547370173